com_migratetojoomla/admin/src/Model/MainModel.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_migratetojoomla
+ *
+ * @copyright   (C) 2013 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\MigrateToJoomla\Administrator\Model;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\MVC\Model\AdminModel;
+use Joomla\CMS\Versioning\VersionableModelTrait;
+
+// phpcs:disable PSR1.Files.SideEffects
+\defined('_JEXEC') or die;
+// phpcs:enable PSR1.Files.SideEffects
+
+/**
+ * Migratetojoomla Component main Model
+ *
+ * @since  1.0
+ */
+class MainModel extends AdminModel
+{
+    use VersionableModelTrait;
+
+    /**
+     * @var    string  The prefix to use with controller messages.
+     * @since  1.0
+     */
+    protected $text_prefix = 'COM_MIGRATETOJOOMLA';
+
+    /**
+     * @var    string  The type alias for this content type.
+     * @since  1.0
+     */
+    public $typeAlias = 'com_migratetojoomla.main';
```

Comment:
Do you have a type registered in the database or is it needed at all?

Reply:
No type registered in database .  It is used by joomla internally I don't know much about it use cases